apps/dolly-frontend/src/main/js/src/components/bestilling/sammendrag/kriterier/BestillingKriterieMapper.js

```diff
@@ -493,6 +496,49 @@ export function mapBestillingData(bestillingData, bestillingsinformasjon) {
 			data.push(sikkerhetstiltakData)
 		}
 
+		if (tilrettelagtKommunikasjon) {
+			const tilrettelagtKommunikasjonData = {
+				header: 'Tilrettelagt Komm.',
+				itemRows: tilrettelagtKommunikasjon.map((item, idx) => {
+					return [
+						{ numberHeader: `Tolk ${idx + 1}` },
+						obj('Talespraak', item.spraakForTaletolk, PersoninformasjonKodeverk.Spraak),
+						obj('Tegnspråk', item.spraakForTegnspraakTolk, PersoninformasjonKodeverk.Spraak),
```

Comment:
Tegnspråk

Reply:
Haha, her hadde jeg faktisk klart å få det til av en eller annen grunn 😂

Reply:
Hahaha, men jeg bare kommenterer uten å lese engang! Beklager for null tillit! 😂